class SectionException implements Exception {
  final String cause;
  const SectionException(this.cause);
}
